### ISSUE TITLE
Pre-Populate Value setting not working on Radio Buttons field

### ISSUE THREAD

**Issue by robmcfadden** [verbb/formie#1254]
### Describe the bug

Setting the "Pre-Populate Value" field in the Settings tab on a Radio Buttons field gives the following error on the frontend:

```Calling unknown method: verbb\formie\fields\formfields\Radio::setPrePopulatedValue()```

This error happens once I attempt to pass in the value of that field as a URL parameter on the frontend form. Using a custom templates path here, but the files are unaltered from what's generated by the plugin.

### Steps to reproduce

1. Create a form with a Radio Buttons field
2. Fill in the "Pre-Populate Value" field in the Settings tab for that field
3. On the frontend, attempt to pass in that value with a URL parameter

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes

### Craft CMS version

Craft Pro 4.3.5

### Plugin version

2.0.20

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.20",
  "...": "..."
}
```

Then run `composer update`.

**Comment by robmcfadden** [comment]
Thanks for the lightning quick turnaround! 🙌

**Comment by engram-design** [comment]
Fixed in [2.0.21](https://github.com/verbb/formie/releases/tag/2.0.21)
